src/components/challengeWidget/challengeWidget.css

```diff
@@ -4,29 +4,33 @@
   background-color: white;
 }
 
-.challengeWidget h2 {
+.challengeWidget,
```

Comment:
These rules will looks like `.challengeWidget h2` without the comma. This is called the [Descended combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Descendant_combinator) in CSS.